core-bundle/contao/drivers/DC_Table.php

```diff
@@ -392,6 +392,8 @@ public function showAll()
 			);
 
 			$objSession->set('CLIPBOARD', $arrClipboard);
+
+			$this->redirect($this->getReferer());
```

Comment:
This needs to be reverted: https://github.com/contao/contao/pull/7055#pullrequestreview-1988503023